docs/runbooks/SSPHighRateRejectedVms.md

```diff
@@ -1,25 +1,33 @@
 # SSPHighRateRejectedVms
+<!--apinnick Nov 2022-->
 
 ## Meaning
 
-This alert indicates that many Virtual Machines were not valid and therefore were not created or not modified.
-It means either a user or a script is continuously trying to create invalid Virtual Machines.
+This alert fires when a large number of invalid virtual machines (VMs) is detected by the Template Validator.
```

Comment:
@apinnick  I think we should add here that it means either a user or a script is continuously trying to create or modify Virtual but since the change is invalid they will not be created/updated.

Reply:
OK, that wasn't clear to me from the original. I'll fix it.